README.md

```diff
@@ -1,2 +1,31 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+**Pseudocode for Fallout 4 hacking game**
+
+	guesses = 4 # Number of guesses used for loop
```

Comment:
The number 4 is hard-coded in quite a few places -- would be better to make this a named constant